backend/src/test/java/com/redhat/cloud/notifications/models/behaviorgroup/CreateBehaviorGroupRequestTest.java

```diff
@@ -0,0 +1,115 @@
+package com.redhat.cloud.notifications.models.behaviorgroup;
+
+import com.redhat.cloud.notifications.routers.models.behaviorgroup.CreateBehaviorGroupRequest;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import javax.validation.Validation;
+import javax.validation.Validator;
+import javax.validation.ValidatorFactory;
+import java.util.UUID;
+
+public class CreateBehaviorGroupRequestTest {
+
+    private static Validator validator;
+
+    /**
+     * Sets up the validator.
+     */
+    @BeforeAll
+    public static void setUp() {
+        try (ValidatorFactory validatorFactory = Validation.buildDefaultValidatorFactory()) {
+            validator = validatorFactory.getValidator();
+        }
+    }
+
+    /**
+     * Tests that a valid request doesn't raise any constraint violations.
+     */
+    @Test
+    void testValidRequest() {
+        final var requestBundleId = new CreateBehaviorGroupRequest();
+        requestBundleId.bundleId = UUID.randomUUID();
+        requestBundleId.displayName = "valid display name";
+
+        final var requestBundleName = new CreateBehaviorGroupRequest();
+        requestBundleName.bundleId = UUID.randomUUID();
```

Comment:
Shouldn't this one be
```java
        requestBundleName.bundleName= // something
```
?

Reply:
You are right, yes! I've fixed the test.